pkgs/development/python-modules/torch/source/default.nix

```diff
@@ -280,7 +291,7 @@ buildPythonPackage rec {
       ./clang19-template-warning.patch
       # fix invalid static cast in XNNPACK
       # https://github.com/google/XNNPACK/issues/7489
-      ./xnnpack-bfloat16.patch
+      # ./xnnpack-bfloat16.patch
```

Comment:
I think we should remove the whole block and the patch file to make this change final.